src/main/java/mate/academy/bookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import mate.academy.bookstore.validation.FieldMatch;
+
+@Getter
+@Setter
+@FieldMatch(first = "password", second = "repeatPassword", message = "Passwords do not match")
+public class UserRegistrationRequestDto {
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```
It's better for String. check other places